app/controllers/statements_controller.rb

```diff
@@ -3,7 +3,6 @@ class StatementsController < ApplicationController
   before_filter :authenticate_user!
   before_filter :check_acting_as
   before_filter :init_account
-  before_filter :init_statement, :except => :index
```

Comment:
Or is `StatementsController#show` even a valid path? I can't quite figure out how to get to it other than direct url manipulation.